packages/angular/.gitignore

```diff
@@ -1,7 +1,4 @@
 # Outputs
-src/**/*.js
-src/**/*.js.map
-src/**/*.d.ts
 dist
```

Comment:
When I created the initial version of the schematic, it was outputing compiled JS files directly into the src folder. I updated the output folder but forgot to update the .gitignore.